script/gen_requirements_all.py

```diff
@@ -432,7 +451,12 @@ def requirements_output() -> str:
         "\n",
         "# Home Assistant Core\n",
     ]
-    output.append("\n".join(core_requirements()))
+
+    requirements = set()
+    requirements.update(core_requirements())
+    requirements.update(gather_entity_platform_requirements())
+
+    output.append("\n".join(sorted(requirements)))
```

Comment:
```suggestion
    output.append("\n".join(sorted(requirements, key=lambda key: key.lower())))
```
pip is case insensitive, so we should be too.

Reply:
Yup that make sense.